README.aspnetcore-build.md

```diff
@@ -45,50 +45,75 @@ The CI Image (`1.0-1.1`) contains both the 1.0 and 1.1 pre-restored packages. It
 
 ## Example Usage
 
-### Build an app with `docker run`
-
-You can use this container to compile your application when it runs. If you use the [Visual Studio tooling](https://blogs.msdn.microsoft.com/webdev/2016/11/16/new-docker-tools-for-visual-studio/) to setup CI/CD to Azure Container Service then this method of using the build container is used.
-
-Run the build container, mounting your code and output directory, and publish your app:
-
-```
-docker run -it -v $(PWD):/app --workdir /app microsoft/aspnetcore-build bash -c "dotnet restore && dotnet publish -c Release -o ./bin/Release/PublishOutput"
-```
+### Build an app with `docker build`
 
-After this has run the application in the current directory will be published to the `bin/Release/PublishOutput` directory.
+With this technique your application is compiled in two stages when you run `docker build`.
+The requires Docker 17.05 or newer.
+Stage 1 compiles your application in `/source`.
+Stage 2 copies the compiled app into `/app` and configures the app to launch your website when the container starts. This does not preserve the source code from stage 1.
```

Comment:
`This does not preserve the source code from stage 1.`  To me this almost makes it sound like the source code is deleted from stage 1.  I live the way [Docker describes multi-stage-builds](https://docs.docker.com/engine/userguide/eng-image/multistage-build/#use-multi-stage-builds) - `leaving behind everything you don't want in the final image`

```
Stage 1 compiles and published the application by using the microsoft/aspnetcore-build image.  Stage 2 copies the published application from Stage 1 into the final image leaving behind all of the source code and tooling needed to build.
```